### PR TITLE
update readme about archival

### PR DESCRIPTION
**Pull request by andy-neuma**
SUMMARY:
* update README with information about archiving this REPO.

### CODE REVIEW

**Review by NMsasa — APPROVED** [review]
looks great

**Review by robertgshaw2-redhat — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,43 +1,10 @@
 # nm-vllm
 
-## Overview
+__THIS REPO HAS BEEN ARCHIVED AS OF SEPTEMBER 2024. NEURAL MAGIC IS STILL RELEASING ENTERPRISE PACKAGES RELATED TO VLLM. OUR RELEASE REPO HAS JUST GONE PRIVATE.__
 
-`nm-vllm` is our supported enterprise distribution of [vLLM](https://github.com/vllm-project/vllm).
+To contribute and to see our contributions to vLLM , visit [vLLM](https://github.com/vllm-project/vllm).
```
Comment (jeanniefinks) [inline README.md:5]:
```suggestion
To contribute and to see our contributions to vLLM, visit [vLLM](https://github.com/vllm-project/vllm).
```

File: README.md [hunk 2]
```diff
@@ -1,43 +1,10 @@
...
-```bash
-MODEL_ID=Qwen/Qwen2-0.5B-Instruct
-docker run --gpus all --shm-size 2g ghcr.io/neuralmagic/nm-vllm-openai:latest --model $MODEL_ID
-```
-
-## Models
+To view the latest releases, benchmarking, models, and evaluations from Neural Magic, visit [nm-vllm-certs](https://github.com/neuralmagic/nm-vllm-certs).
 
 Neural Magic maintains a variety of optimized models on our Hugging Face organization profiles:
 - [neuralmagic](https://huggingface.co/neuralmagic)
```
Comment (jeanniefinks) [inline README.md:10]:
```suggestion
- [neuralmagic](https://huggingface.co/neuralmagic)

To learn more about nm-vllm Enterprise, visit the [nm-vllm product page](https://neuralmagic.com/nm-vllm/).
```

Comment (jeanniefinks) [reply README.md:10]: @andy-neuma see edits above

**Review by jeanniefinks — CHANGES_REQUESTED** [review]
Added context for users to learn more about nm-vllm enterprise; fixed minor typo

File: README.md (hunk 1 above)
Comment (andy-neuma) [reply README.md]: good catch!

File: README.md (hunk 2 above)
Comment (andy-neuma) [reply README.md:12]: cool, i inserted the reference to our product page higher up.

**Review by jeanniefinks — APPROVED** [review]

**Pull request merged**